generator/Dockerfile

```diff
@@ -1 +1 @@
-FROM golang:1.6-onbuild
+FROM golang:1.8-onbuild
```

Comment:
onbuild is deprecated. let's use the 1.8 image directly, but then add the wrapper commands to our dockerfile:
https://github.com/docker-library/golang/blob/132cd70768e3bc269902e4c7b579203f66dc9f64/1.8/onbuild/Dockerfile